cms/templates/cms/program_page.html

```diff
@@ -104,11 +104,37 @@
     </div>
   </div>
 
+  {% if page.faculty_members.count > 0 %}
+    <div class="faculty-section">
+      <h3 class="title">Faculty</h3>
+      {% if page.faculty_description|length > 0 %}
+        <p class="description">{{ page.faculty_description }}</p>
+      {% endif %}
+      <div id="faculty-carousel" class="single-column-wide">
```

Comment:
A future version of `ReactDOM.render` may leave the HTML below in place so the behavior of replacing the contents of the container is not guaranteed. See the red box: https://facebook.github.io/react/docs/top-level-api.html#reactdom.render

Instead can we populate a JS array with the faculty data? This would also make `FacultyCarousel` easier to test